### PR TITLE
Pull request for issue #142 - Backend

### PR DESCRIPTION
**Pull request by umutuz7**

### CODE REVIEW

File: Web-App/src/main/resources/migrations/V1.0.3__create_violations.sql [hunk 1]
```diff
@@ -3,8 +3,8 @@ CREATE TABLE `violation` (
   `user_id` int(11) unsigned NOT NULL,
   `description` text,
   `date` date NOT NULL,
+  `modification_date` date DEFAULT NULL,
   `title` varchar(255) NOT NULL DEFAULT '',
-  `severity_rate` int(4) unsigned NOT NULL,
```
Comment (omermtemel) [inline Web-App/src/main/resources/migrations/V1.0.3__create_violations.sql]: This change should be in another migration in which severity_date column is deleted and modficationDate added.

ALTER TABLE `violation` DELETE COLUMN severity_rate;
ALTER TABLE `violation` ADD COLUMN modification_date DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP

File: Web-App/src/main/resources/migrations/V1.0.3__create_violations.sql [hunk 2]
```diff
@@ -4,11 +4,11 @@ CREATE TABLE `violation` (
   `description` text,
   `date` date NOT NULL,
   `title` varchar(255) NOT NULL DEFAULT '',
-  `severity_rate` int(4) unsigned NOT NULL,
+  `severity_rate` int(4) unsigned NOT NULL,  `title` varchar(255) NOT NULL DEFAULT '',
```
Comment (omermtemel) [inline Web-App/src/main/resources/migrations/V1.0.3__create_violations.sql]: There is somethogn wrong

**Comment by omermtemel** [comment]
You should add new migration file into commit

**Comment by umutuz7** [comment]
I fixed the problems. Please ignore the message of last commit (Best practice format is applied to API)

**Comment by omermtemel** [comment]
LGTM

**Pull request merged**